@here/harp-mapview/lib/MapView.ts

```diff
@@ -1222,7 +1215,7 @@ export class MapView extends THREE.EventDispatcher {
      */
     setFovCalculation(fovCalculation: FovCalculation) {
         this.m_options.fovCalculation = fovCalculation;
-        this.calculateFocalLength(this.m_renderer.getSize().height);
+        this.calculateFocalLength(this.m_renderer.getSize(new THREE.Vector2()).height);
```

Comment:
Could we have a unique vector2 in the file that is reused there everytime instead?